airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -181,7 +181,7 @@
     # [START howto_operator_gcp_transfer_list_operations]
     list_operations = GcpTransferServiceOperationsListOperator(
         task_id="list_operations",
-        filter={
+        request_filter={
```

Comment:
This is breaking compatibility change. You should provide a backward compatibility mechanism and add a note in UPDATING.md

Reply:
I am not sure if this purist change it's worth it. We can disable `redefined-builtin` inspection here. What, do you think?

Reply:
I think that this problem is worth solving. Other operators have similar problems and they have been resolved there.Backward compatibility will be easy to provide here. If you want, then on Monday I will show you how to do it.